cmd/kubeadm/app/cmd/token.go

```diff
@@ -132,13 +133,15 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 				return err
 			}
 
-			return RunCreateToken(out, client, cfgPath, cfg, printJoinCommand, kubeConfigFile)
+			return RunCreateToken(out, client, cfgPath, cfg, printJoinCommand, certificateKey, kubeConfigFile)
 		},
 	}
 
 	options.AddConfigFlag(createCmd.Flags(), &cfgPath)
 	createCmd.Flags().BoolVar(&printJoinCommand,
 		"print-join-command", false, "Instead of printing only the token, print the full 'kubeadm join' flag needed to join the cluster using the token.")
+	createCmd.Flags().StringVar(&certificateKey,
+		"certificate-key", "", "When used together with --print-join-command, print the full 'kubeadm join' flag needed to join the cluster as a master. You must create the certificate key using 'kubeadm upload-certs --upload-certs' yourself.")
```

Comment:
under /cmd there is an options package that should have "certificate-key" as a constant.

also please reword the description to:
> When used together with '--print-join-command', print the full 'kubeadm join' flag needed to join the cluster as a control-plane. To create a new certificate key you must use 'kubeadm init phase upload-certs --upload-certs'."

Reply:
I can't find the certificate-key constant, just client-key and client-certificate, which I believe are both a different thing?

Reply:
should be this one https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/options/constant.go#L132

Reply:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/options/constant.go#L132

Reply:
Not sure how I managed to miss that, thanks!